spec/tapioca/compilers/dsl/action_controller_helpers_spec.rb

```diff
@@ -217,5 +217,19 @@ class UserController::HelperProxy < ::ActionView::Base
 
       assert_equal(expected, rbi_for(:UserController, files))
     end
+
+    it("generates an empty rbi file for anonymous classes") do
```

Comment:
With my recommended change, this would become a simpler `gather` phase test like:
```
    it("ignores anonymous subclasses of ActionController") do
      content = <<~RUBY
        UserController = Class.new(ActionController::Base)
      RUBY

      assert_equal([], constants_from(content))
    end
```

Reply:
This also does not necessitate the usage of `T.unsafe`.

Reply:
Lol, I just noticed that I gave the example with the constant assigned a name again 😂